src/plugins/global-settings/colors/index.js

```diff
@@ -0,0 +1,360 @@
+import './responsive-color-palette-change'
+
+/**
+ * Internal dependencies
+ */
+import { GlobalSettingsColorPicker } from '../components'
+import { replaceGlobalColorAttributes, updateFallbackColorAttributes } from './util'
+
+/**
+ * External dependencies
+ */
+import { i18n } from 'stackable'
+import { PanelAdvancedSettings } from '~stackable/components'
+import {
+	isEqual, find, omit,
+} from 'lodash'
+import md5 from 'md5'
+import rgba from 'color-rgba'
+
+/**
+ * Wordpress dependencies
+ */
+import {
+	addFilter, addAction, doAction,
+} from '@wordpress/hooks'
+import {
+	Fragment, useState, useEffect,
+} from '@wordpress/element'
+import { __ } from '@wordpress/i18n'
+import domReady from '@wordpress/dom-ready'
+import { loadPromise, models } from '@wordpress/api'
+import {
+	dispatch, useSelect, select,
+} from '@wordpress/data'
+
+addFilter( 'stackable.global-settings.inspector', 'global-settings/global-colors', output => {
+	const [ isPanelOpen, setIsPanelOpen ] = useState( true )
+	const [ disableReset, setDisableReset ] = useState( true )
```

Comment:
Initial state of disabling should be false, so if something wrong happened, the user shouldn't be able to reset the colors